.github/workflows/ci.yml

```diff
@@ -76,7 +80,7 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: doc
-          args: --all --document-private-items
+          args: --workspace --document-private-items --all-features
```

Comment:
Not sure why we need `workspace` when this is a single crate. Could you elaborate on that?

Reply:
`--all` is the deprecated form of `--workspace` so I was just bringing it up to date.

Also, note that I was able to push changes to the CI which is not something non-contributors should be allowed to do. You should probably change the permissions settings so that only core contributors can change actions files.